Adding Summary Document Requirements (#412)
w3c/wot-binding-templates: registry-requirements.md

6. The binding document linked in the registry entry SHOULD be open to read, use, and implement, but that is not required for the document be added to the registry.
  - See also https://github.com/w3c/wot-binding-templates/issues/393
7. Reviewer MUST have access to the binding document and to the protocol or media type specification (what the binding specifies)

> 8. The submitter MUST fill the GitHub form provided by the custodian for the custodian to generate the summary document, which is hosted by the custodian together with the registry. This form contains the following:

Reviewer: ```suggestion
8. The submitter MUST fill the GitHub form provided by the custodian to generate the summary document, which is hosted by the custodian together with the registry. This form contains the following:
```

Somehow doubled!?

Author: yup makes sense

  - Abstract - It MUST contain an abstract with the following information:

>     - What the binding's content is about, e.g. what this protocol is.
>     - Who should use it
>     - For which purpose should it be used, e.g. monitoring, process control. This SHOULD use terminology of the submitter, i.e. the custodian does not provide definitions for this.

Reviewer 2: ```suggestion
    - What is the content of the binding about, e.g., what is this protocol?
    - Who should use it?
    - For what purpose(s) should it be used, e.g., monitoring, process control? This SHOULD use terminology of the submitter, i.e., the custodian does not provide definitions for this.
```

  -  Examples - It SHOULD contain examples (can be one) TDs or TMs demonstrating the use of the binding

>   -  It MUST contain Access/Usage restrictions about the binding, protocol, implementation etc. using the terminology and/or documents of the submitter. A non-exhaustive list of examples of restrictions: 

Reviewer 2: ```suggestion
  -  It MUST contain Access/Usage restrictions about the binding, protocol, implementation, etc., using the terminology and/or documents of the submitter. A non-exhaustive list of examples of restrictions: 
```

    - Reading the binding document
    - Reading the protocol specification
    - Implementing a non-commercial device/Thing
    - Implementing a non-commercial Consumer application/driver

>     - Conditions for commercial use: E.g. building a commercial product with the binding

Reviewer 2: ```suggestion
    - Conditions for commercial use, e.g., building a commercial product with the binding
```

>     - Making a statement about your product's supporting that binding

Reviewer 2: ```suggestion
    - Making a statement about a product's support for that binding
```
